plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1454,6 +1512,9 @@ function( $subscription ) use ( $site_id ) {
 			'product_id'
 		);
 
+		// Track installed subscriptio ids to avoid duplicate entries for inactive subsriptions.
+		$installed_subscription_ids = array();
```

Comment:
```suggestion
		// Track installed subscription ids to avoid duplicate entries for inactive subsriptions.
		$installed_subscription_ids = array();
```